themes/app/templates/Includes/Footer.ss

```diff
@@ -1,45 +1,45 @@
 <footer>
     <div class="container footer__partners">
         <div class="row">
-            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6 ">
+            <div class=" col-lg-2 col-md-3 col-sm-6 col-6 ">
                 Our Partners
             </div>
         </div>
         <div class="row">
-            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6">
+            <div class=" col-lg-2 col-md-3 col-sm-6 col-6">
                     $SiteConfig.SponsorLogo1
             </div>
-            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6">
+            <div class=" col-lg-2 col-md-3 col-sm-6 col-6">
                     $SiteConfig.SponsorLogo2
             </div>
-            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6">
+            <div class=" col-lg-2 col-md-3 col-sm-6 col-6">
                     $SiteConfig.SponsorLogo3
             </div>
-            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6">
+            <div class=" col-lg-2 col-md-3 col-sm-6 col-6">
                     $SiteConfig.SponsorLogo4
             </div>
-            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6">
+            <div class=" col-lg-2 col-md-3 col-sm-6 col-6">
                     $SiteConfig.SponsorLogo5
             </div>
-            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6">
+            <div class=" col-lg-2 col-md-3 col-sm-6 col-6">
                     $SiteConfig.SponsorLogo6
             </div>
         </div>
     </div>
     <div class="container-fluid footer__info">
         <div class="row">
-            <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
+            <div class="col-lg-3 col-md-6 col-sm-12">
                 $SiteConfig.PhoneNumber </br>
                 $SiteConfig.Email</br>
                 $SiteConfig.Facebook
             </div>
-            <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
+            <div class="col-lg-3 col-md-6 col-sm-12">
                 $SiteConfig.FooterSentence1
             </div>
-            <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12 order-md-12 order-lg-1">
+            <div class="col-lg-3 col-md-6 col-sm-12 order-md-12 order-lg-1">
```

Comment:
Im not too sure what `order-md-12` and `order-lg-1` does, are they needed? If you remove them does it have an impact?

Reply:
It impacts the ordering which the columns appear when the screen is shrunk. I had to use them to position the components in the right order when in mobile, as the order is different in larger screens.

Reply:
Oh right interesting, I'll have to see this in person but should be ok for now :)